### PR TITLE
EdgeR: Minor fixes and add option to output Rscript

### PR DESCRIPTION
**Pull request by mblue9**

### CODE REVIEW

**Comment by nsoranzo** [comment]
It seems the small differences in floats is a problem here too.

File: tools/edger/edger.xml [hunk 1]
```diff
@@ -278,8 +287,16 @@ cp '$outReport.files_path'/*.tsv output_dir/
             </repeat>
             <param name="normalisationOption" value="TMM" />
             <output_collection name="outTables" count="2">
-                <element name="edgeR_Mut-WT" ftype="tabular" file="edgeR_Mut-WT.tsv" />
-                <element name="edgeR_WT-Mut" ftype="tabular" file="edgeR_WT-Mut.tsv" />
+                <element name="edgeR_Mut-WT" ftype="tabular" >
+                    <assert_contents>
```
Comment (nsoranzo) [inline tools/edger/edger.xml]: What about matching also the header line (for each tabular file)?

Comment (nsoranzo) [reply tools/edger/edger.xml]: Can you add `<has_text_matching expression="GeneID.*logFC.*logCPM.*F.*PValue.*FDR" />` here too?

Comment (mblue9) [reply tools/edger/edger.xml]: done

**Comment by bgruening** [comment]
Thanks @mblue9!

**Pull request merged**